<launch>

  <!-- Launch Sick Microscan3 Ros Driver Node -->
  <node pkg="sick_microscan3_ros_driver" type="sick_microscan3_ros_driver" name="sick_microscan3_ros_driver" output="screen">
    <!-- <param name="sensor_ip" type="string" value="111.111.111.111" /> -->
  </node>


  <node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" />
</launch>

